src/nationalcareers_toolkit/assets/src/frontend/sass/compui/skills-toolkit.scss

```diff
@@ -6,49 +6,10 @@
 
 /* CUSTOM STYLES FOR ACCORDION Non Javascript   */
 body {
```

Comment:
I think we should fully revert all the accordion related changes to the skills-toolkit.scss so that it only contains the course-list rules as it was before today?